scripts/build_example

```diff
@@ -5,13 +5,13 @@ readonly SCRIPTS_DIR="$(dirname "$0")"
 source "${SCRIPTS_DIR}/common"
 
 application_language="rust"
-client_language="cpp"
+client_variant="bazel"
 compilation_mode='fastbuild'
 docker_config=''
 while getopts "e:a:c:di:h" opt; do
   case "${opt}" in
     h)
-      echo -e "Usage: ${0} [-h] [-a rust|cpp] [-c rust|cpp] [-i base|logless] [-d] -e EXAMPLE
+      echo -e "Usage: ${0} [-h] [-a rust|cpp] [-c cargo|bazel] [-i base|logless] [-d] -e EXAMPLE
```

Comment:
Looks like there is a npm build step at line 122, why not include npm in the help text?

Reply:
Added that. :) For now I've been using this draft PR as a way to run/test CI in the background. 

We only conditionally call the `build_example` step inside of run_example, and I wanted to check which things would break if I moved `npm` into the variant, but it's not ready yet.